packages/rum-core/src/domain/view/viewMetrics/trackCumulativeLayoutShift.spec.ts

```diff
@@ -37,27 +42,27 @@ describe('trackCumulativeLayoutShift', () => {
   })
 
   afterEach(() => {
+    cleanup()
     if (originalSupportedEntryTypes) {
       Object.defineProperty(PerformanceObserver, 'supportedEntryTypes', originalSupportedEntryTypes)
     }
   })
 
   it('should be initialized to 0', () => {
-    setupBuilder.build()
-
     expect(clsCallback).toHaveBeenCalledOnceWith({ value: 0 })
   })
 
   it('should be initialized to undefined if layout-shift is not supported', () => {
+    // stop the previous tracking from beforeEach
+    cleanup()
```

Comment:
**💬 suggestion:** ‏same here, also you could have `startCLSTracking({ isLayoutShiftSupported: false })`